go.mod

```diff
@@ -1,9 +1,8 @@
 module github.com/golang-migrate/migrate/v4
 
 require (
-	cloud.google.com/go v0.53.0 // indirect
-	cloud.google.com/go/spanner v1.2.0
-	cloud.google.com/go/storage v1.5.0
+	cloud.google.com/go/spanner v1.5.0
```

Comment:
Only this line was changed manually, other changes are from running "go get"

Reply:
Could you also run `go mod tidy`?

Reply:
Done, though still a big diff